src/auth-service/utils/date.js

```diff
@@ -1,3 +1,5 @@
+const { logElement, logObject, logText } = require("@utils/log");
```

Comment:
this was causing a runtime error when you query user logs using `startTime` and/or `endTime` query parameters